agent/agent.go

```diff
@@ -491,6 +499,25 @@ func (a *agent) setLifecycle(ctx context.Context, state codersdk.WorkspaceAgentL
 	}
 }
 
+// fetchServiceBannerLoop fetches the service banner on an interval.
```

Comment:
Perhaps we should add a note here that this won't immediately fetch anything and that it's expected to be primed elsewhere? (This way the intent is clear.)

Reply:
I like this idea.